TODO.org

```diff
@@ -1,2 +1,28 @@
 * CH01-02
 - Substituir o link ao capítulo de Suporte de Editores de Texto por um link que funcione.
+
+* Capítulos faltantes
```

Comment:
Senti falta de colocar como TODO em cada, mas dai aq vai ficar todos e vamos deletar os q fizermos ou vamos colocar DONE?

Reply:
Se for colocar DONE era bom colocar tbm os q ja foram feitos como DONE

Reply:
Ideia de jerico minha. Arquivo deletado. É melhor colocar a única coisa importante dele numa issue.